Library/Homebrew/test/formulary_spec.rb

```diff
@@ -383,6 +384,10 @@ def formula_json_contents(extra_items = {})
       before do
         ENV.delete("HOMEBREW_NO_INSTALL_FROM_API")
 
+        # avoid unnecessary network calls
+        allow(Homebrew::API::Formula).to receive_messages(all_aliases: {}, all_renames: {})
+        allow(CoreTap.instance).to receive(:tap_migrations).and_return({})
```

Comment:
Might be nicer to ensure some correct files are written on disk instead of mocking.

Reply:
Out of laziness, I'm going to skip that for now.